.prettierrc

```diff
@@ -1,7 +1,7 @@
-printWidth: 80
-singleQuote: true
+printWidth: 140
+singleQuote: false
```

Comment:
Please don't :'(

Reply:
What? you wanna keep single quotes? But it doesn't go with any other project we have currently.

Reply:
YES! Death to single quotes!!! Muahahahaha 

just joking I don't care 👅

Reply:
This is very helpful when you're moving a lot of code around. All our old code is written with double-quotes. It means you're gonna deal with a lot of unnecessary linting issues.